### ISSUE TITLE
Feature request: extensions for non-photorealistic shading in combination with global illumination and for global illumination in 2.5D games?

### ISSUE THREAD

**Issue by ADormant** [KhronosGroup/Vulkan-Docs#2032]
1. Currently cel shading and other non-photorealistic shading methods can't be used in combination with global illumination. Could you add Vulkan extensions allowing for non-photorealistic shading directly on textures and in combination with the global illumination?
2. Another useful thing would be an extension for for global illuminations in 2.5D side-scrolling, top-down, and isometric/parallel projection games.

**Comment by HansKristian-Work** [comment]
I'm not sure how this relates to Vulkan. These are all high level concepts that are not related to the underlying graphics API at all.

**Comment by krOoze** [comment]
I am also not sure how Vulkan's ray tracing feature precludes non-photorealistic shading. The shaders are programable, and Vulkan does not prescribe which color you put into a pixel.

**Comment by oddhack** [comment]
@ADormant can this issue be closed?

**Comment by oddhack** [comment]
Er, let me rephrase since I accidentally closed it rather than leaving that for you to do. As @HansKristian-Work explained, this functionality is not something that would be provided through a low-level API like Vulkan, but something you might use Vulkan to implement through shaders and middleware libraries. I'm going to reopen but encourage you to close this as this isn't functionality the Vulkan WG is going to provide in the forseeable future.

**Comment by oddhack** [comment]
Closing per my previous comment.
